app/controllers/avo/actions_controller.rb

```diff
@@ -73,12 +73,15 @@ def respond
               turbo_stream.close_action_modal,
               turbo_stream.flash_alerts
             ]
+          when :navigate_to_action
+            frame_id = Avo::ACTIONS_TURBO_FRAME_ID
+            src, _ = @response[:action].link_arguments(resource: @action.resource, **@response[:navigate_to_action_args])
+
+            render turbo_stream: turbo_stream.turbo_frame_set_src(frame_id, src)
           when :redirect
-            # Turbo redirect to the path
             render turbo_stream: turbo_stream.redirect_to(
               Avo::ExecutionContext.new(target: @response[:path]).handle,
-              nil,
-              @response[:redirect_args][:turbo_frame],
+              turbo_frame: @response[:redirect_args][:turbo_frame],
               **@response[:redirect_args].except(:turbo_frame)
```

Comment:
Not sure if this modification follows the `redirect_to` arguments syntax:
https://github.com/marcoroth/turbo_power-rails/blob/80f9d1a8b8aa43200557d3210d0b2cabe0700c47/lib/turbo_power/stream_helper.rb#L239